ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -115,6 +115,61 @@ export async function getEditorScreenshot(
 ) {
   return await page.locator('[class*="App-module_canvas"]').screenshot(options);
 }
+const DECIMAL_PLACES = 1000;
+
+function roundToThirdDecimal(num: number) {
+  return Math.round(num * DECIMAL_PLACES) / DECIMAL_PLACES;
+}
+export async function compareFiles(file1Paths: string[], file2Paths: string[]) {
+  try {
+    const [file1Content, file2Content] = await Promise.all([
+      readFileContents(file1Paths[0]),
+      readFileContents(file2Paths[0]),
+    ]);
+
+    const file1Lines = file1Content.split('\n');
+    const file2Lines = file2Content.split('\n');
+
+    // Check if the lengths of file1Lines and file2Lines differ
+    if (file1Lines.length !== file2Lines.length) {
```

Comment:
I guess, we need to remove this check, because sometimes coordinates may have different amount of digits. For example, in one file `12.34` and in another `12.345`. So, this check will say that files are not identical